src/transformers/models/big_bird/modeling_big_bird.py

```diff
@@ -1055,6 +1090,16 @@ def _get_rand_attn_plan(from_seq_length, from_block_size, num_rand_blocks):
 
         return plan_from_length, plan_num_rand_blocks
 
+    def _bigbird_block_rand_mask_fast(self, 
+            from_seq_length, to_seq_length, from_block_size, to_block_size, num_rand_blocks, last_idx, n_heads
```

Comment:
Can you add similar docstring as the original function? This will help understanding.

Reply:
fixed by https://github.com/huggingface/transformers/pull/21499/commits/be847de12f092e8b8645966a496dd7891b2eb31d